packages/mosaic/timepicker/timepicker.spec.ts

```diff
@@ -594,5 +596,37 @@ describe('McTimepicker', () => {
                     expect(inputElementDebug.nativeElement.value).toBe('23:59', 'Arrow-Down key decrement not working');
                 });
         });
+
+        it('Manual keyboard input digit-by-digit', () => {
+            return fixture.whenStable()
+                .then(() => {
+                    dispatchFakeEvent(inputElementDebug.nativeElement, 'focus');
```

Comment:
Do you really need dispatch this events ?

Reply:
No, removed.